extensions/kubectl-cli/src/extension.ts

```diff
@@ -226,11 +226,9 @@ export async function activate(extensionContext: extensionApi.ExtensionContext):
 
   // Push the CLI tool as well (but it will do it postActivation so it does not block the activate() function)
   // Post activation
-  setTimeout(() => {
-    postActivate(extensionContext, kubectlDownload).catch((error: unknown) => {
-      console.error('Error activating extension', error);
-    });
-  }, 0);
+  postActivate(extensionContext, kubectlDownload).catch((error: unknown) => {
+    console.error('Error activating extension', error);
+  });
```

Comment:
Why removing the setTimeout ?

It was used as per the comment above to avoid blocking the activate function, reason to remove ?

Reply:
I would say that as there was no await, it was not blocking anyway the activate function
but yes it seems not related to the scope of this PR

Reply:
I have a preference for the promise without await than the setTimout anyway